runtime/compiler/trj9/optimizer/JProfilingRecompLoopTest.cpp

```diff
@@ -52,6 +52,21 @@ int32_t TR_JProfilingRecompLoopTest::getLoopNestingDepth(TR::Compilation *comp,
    return nestingDepth;
    }
 
+/**
+ * A utility function that iterates through the TR::list of byte code info and checks if passed bytecode info exists in list
+ * by checking bytecode index and caller index of the bytecode info
+ */ 
+bool
+TR_JProfilingRecompLoopTest::isByteCodeInfoInCurrentTestLocationList(TR_ByteCodeInfo &bci, TR::list<TR_ByteCodeInfo, TR::Region&> addedLocationBCIList)
```

Comment:
you should pass that list by reference rather than value to save copying the list.